luigi_pipeline/lib/model/sv_mt_schema.py

```diff
@@ -74,15 +81,15 @@ def gnomad_svs_af(self):
 
     @row_annotation(name='gnomad_svs_AC')
     def gnomad_svs_ac(self):
-        return self.mt.info.gnomAD_V2_AC_AF
+        return hl.int32(self.mt.info.gnomAD_V2_AC)
```

Comment:
I ran `mt.aggregate_rows(hl.agg.counter(mt.info.gnomAD_V2_AC))` and didn't see any non-integers (though I didn't look very thoroughly).  This'll throw if we encounter a float, so I think we're good as is here.

Reply:
ah...

I did 
```
hl.eval(hl.int32('1.1'))
```